apps/chat/bots.py

```diff
@@ -407,6 +427,25 @@ def _save_message_to_history(
                 chat_message.create_and_add_tag(tag_value, self.session.team, category or "")
         return chat_message
 
+    def synthesize_voice(self) -> tuple["VoiceProvider", "SyntheticVoice"] | None:
+        from apps.experiments.models import SyntheticVoice
+        from apps.service_providers.models import VoiceProvider
+
+        voice_provider = (
+            VoiceProvider.objects.get(id=self.voice_provider_id)
+            if getattr(self, "voice_provider_id", None) is not None
```

Comment:
nit: `getattr` isn't necessary here since these fields are declared on the class during init.
```suggestion
            if self.voice_provider_id is not None
```